README.md

```diff
@@ -33,8 +33,25 @@ Example:
 ```js
 import atomLinter from 'atom-linter'
 
-const myLinter = {
+const myFileLinter = {
   // ...
+  scope: 'file',
+  async lint(textEditor) {
+    const textEditorPath = textEditor.getPath()
+    const output = atomLinter.exec('myprogram', ['parameter1', 'parameter2'], { uniqueKey: `my-linter:${textEditorPath}` })
```

Comment:
Since every bit of the code between these is duplicated, why not just show both versions of this line with a comment describing which is which?